source/inbo_rapport_delen/index.Rmd

```diff
@@ -1,23 +1,31 @@
 --- 
 title: "Voorbeeldrapport met INBOmd"
-author: 
-  - firstname: Thierry
-    name: Onkelinx
+author:
+  - name:
+      given: Thierry
+      family: Onkelinx
     email: thierry.onkelinx@inbo.be
     orcid: 0000-0001-8804-4216
-shortauthor: Onkelinx, T.
-corresponding: thierry.onkelinx@inbo.be
+    affiliation: Research Institute for Nature and Forest (INBO)
+    corresponding: true
 reviewer:
-  - firstname: Floris
-    name: Vanderhaeghe
+  - name:
+      given: Floris
+      family: Vanderhaeghe
     email: floris.vanderhaeghe@inbo.be
+    affiliation: Research Institute for Nature and Forest (INBO)
     orcid: 0000-0002-6378-6229
 github-repo: inbo/inbomd-examples
+funder: Research Institute for Nature and Forest (INBO) 
+rightsholder: Research Institute for Nature and Forest (INBO)
+community: inbo
+keywords: corporate identity; example
+publication_type: report
```

Comment:
It would be nice to know what the possibilities are here. Does this refer to the stardard publication types in bibTeX?

Reply:
```
c(
    "annotationcollection", "book", "section", "conferencepaper",
    "datamanagementplan", "article", "patent", "preprint", "deliverable",
    "milestone", "proposal", "report", "softwaredocumentation",
    "taxonomictreatment", "technicalnote", "thesis", "workingpaper", "other"
  )
```
`checklist::bookdown_zenodo()` will list the available options when it is missing.